starphys/spartanstay
# Backend webserver proof of life

Set up file structure, created user model, ran the server and it listens on port 8080.

Testing: 
1. Run the program from IntelliJ
2. Navigate to localhost:8080
3. Error page should be shown
![image](https://user-images.githubusercontent.com/84964464/190929157-1149d7b7-9997-4075-89f6-16e744e34979.png)
